src/main/scala/com/mwz/sonar/scala/junit/JUnitSensor.scala

```diff
@@ -64,22 +65,6 @@ final class JUnitSensor(
     val reports: List[Path] = reportPaths(config)
     log.debug(s"The JUnit report paths are: ${reports.mkString("[", ",", "]")}.")
 
-    // Get test input files
-    val inputFiles: Iterable[InputFile] =
-      context.fileSystem
-        .inputFiles(
-          context.fileSystem.predicates.and(
-            context.fileSystem.predicates.hasLanguage(Scala.LanguageKey),
-            context.fileSystem.predicates.hasType(InputFile.Type.TEST)
-          )
-        )
-        .asScala
-
-    if (inputFiles.nonEmpty)
-      log.debug(s"Input test files: \n${inputFiles.mkString(", ")}")
-    else
-      log.warn(s"No test files found for module ${context.module.key}.")
-
```

Comment:
Do you think we don't need this anymore?

Reply:
Oh, sorry. Forgot to ask, and end up deleting it anyways.
I did not understand what was the intended use of that.
The `inputFile` are only used for the logs, and I remember there were some similar logs, like empty test directories.

Care to explain what was the use of this?, If you consider it important I can add it again.

Reply:
@mwz So, should I restore this?

Reply:
Yes, I think so. This will log warnings when we don't find any test files in the module, so it's useful - a gentle reminder that you should have tests in your project ;)